src/actions/socialAuthActions.js

```diff
@@ -0,0 +1,46 @@
+import { RECEIVE_DATA, FETCHING, FETCH_FAILED } from './actionTypes';
+
+export function FacebookAuth(data) {
+  return data;
```

Comment:
Are there no action types for these 3 actions?

Reply:
Are these action types?

Reply:
They are action types.

Reply:
I meant are they actions? If they are actions, then they should have an action type they return.

Reply:
I believe I fixed that as per @salma-nyagaka comment below